android/res/values-uk/strings.xml

```diff
@@ -85,42 +85,42 @@
   <string name="msg_share_explanation">Ви можете ділитись даними, скануючи штрих-код з вашого екрану іншим телефоном.</string>
   <string name="msg_share_text">Або введіть текст</string>
   <string name="msg_sure">Ви впевнені?</string>
-  <string name="msg_unmount_usb">Нажаль, SD-карта недоступна.</string>
+  <string name="msg_unmount_usb">На жаль, SD-карта недоступна.</string>
   <string name="preferences_actions_title">Коли штрих-код знайдено\u2026</string>
   <string name="preferences_auto_focus_title">Використовувати автофокус</string>
-  <string name="preferences_auto_open_web_title">Відкрийте веб-сторінок автоматично</string>
+  <string name="preferences_auto_open_web_title">Відкрити веб-сторінку</string>
   <string name="preferences_bulk_mode_summary">Постійно сканувати та зберігати штрих-коди</string>
   <string name="preferences_bulk_mode_title">Масове сканування</string>
   <string name="preferences_copy_to_clipboard_title">Скопіювати до буферу обміну</string>
   <string name="preferences_custom_product_search_summary" formatted="false">Заміни: %s = вміст, %f = формат, %t = тип</string>
   <string name="preferences_custom_product_search_title">URL користувацького пошуку</string>
   <string name="preferences_decode_1D_industrial_title">Промислові 1D</string>
-  <string name="preferences_decode_1D_product_title">1D продукту</string>
-  <string name="preferences_decode_Aztec_title">Ацтекскій</string>
+  <string name="preferences_decode_1D_product_title">Продуктові 1D</string>
+  <string name="preferences_decode_Aztec_title">Розпізнавати Aztec коди</string>
   <string name="preferences_decode_Data_Matrix_title">Розпізнавати Data Matrix</string>
   <string name="preferences_decode_PDF417_title">PDF417 (β)</string>
   <string name="preferences_decode_QR_title">Розпізнавати QR-коди</string>
   <string name="preferences_device_bug_workarounds_title">Виправлення помилок пристрою</string>
-  <string name="preferences_disable_barcode_scene_mode_title">Немає режимі штрих-коду сцени</string>
+  <string name="preferences_disable_barcode_scene_mode_title">Немає режиму штрих-коду сцени</string>
   <string name="preferences_disable_continuous_focus_summary">Стандартний режим фокусування</string>
-  <string name="preferences_disable_continuous_focus_title">Відклюти безперервне фокусування</string>
-  <string name="preferences_disable_exposure_title">Відключити експозицію</string>
+  <string name="preferences_disable_continuous_focus_title">Вимкнути постійне фокусування</string>
+  <string name="preferences_disable_exposure_title">Вимкнути експозицію</string>
   <string name="preferences_disable_metering_title">Немає обліку</string>
+  <string name="preferences_front_light_summary">Покращує розпізнавання при поганому освітлення, але може викликати відблиски. Працює не на всіх пристроях.</string>
```

Comment:
Would you mind moving these two strings back down by 3 lines? that keeps them in sorted order.